mentoring/pairing/models.py

```diff
@@ -22,13 +22,13 @@ def __str__(self):
         Participant,
         related_name='learner_pairing',
         on_delete=models.RESTRICT,
-        limit_choices_to={"role": Participant.LEARNER},
+        limit_choices_to={"is_learner": True},
     )
     mentor = models.ForeignKey(
         Participant,
         related_name='mentor_pairing',
         on_delete=models.RESTRICT,
-        limit_choices_to={"role": Participant.MENTOR},
+        limit_choices_to={"is_mentor": True},
```

Comment:
Side note: model-level enforcement of ``learner != mentor`` is tricky. The method [clean](https://docs.djangoproject.com/en/3.1/ref/models/instances/#django.db.models.Model.clean) can have the logic, and will be called when you save objects in forms like the Django admin. It will not be called if you call ``save`` directly in code.

Reply:
Oops, I moved this down and forgot to delete this one. Resolving.